CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [1.1.14] - 2020-01-19
+* Removed identity core classes from public api (#1158).
+* Fixed possible deadlock caused by thread explosion (#1175)
+* Added pipeline configuration to generate framework for SPM & automate MSAL release (#1194)
+* Extend iOS background tasks to silent and interactive requests.
+
```

Comment:
Did you want to include MSIDLRUCache in this release? 
I think you might also want to mention FRT/RT order flip that Kai did.

Reply:
Thanks, included FRT/MRRT order flip. MSIDLRUCache is included in the common core changelog (since it's not externally usable for MSAL developers).